doc/operation-and-maintenance/known-issues.md

```diff
@@ -14,18 +14,22 @@ You have to iterate over all rooms in the DB (`muc_light_rooms` table) and add m
 Every option is inserted as a separate row and is stored as plain text, so it should be straightforward.
 
 Let's say you were using the following config:
-
-```
-{config_schema, [
-                 "roomname",
-                 "subject",
-                 "background",
-                 "notification_sound"
-                ]},
-{default_config, [
-                  {"roomname", "The room"},
-                  {"subject", "Chit-chat"}
-                 ]}
+```toml
+  [[modules.mod_muc_light.config_schema]]
+    field = "roomname"
+    value = "The room"
+
+  [[modules.mod_muc_light.config_schema]]
+    field = "subject"
+    value = "Chit-chat"
+
+  [[modules.mod_muc_light.config_schema]]
+    field = "background"
+    value = ""
+
+  [[modules.mod_muc_light.config_schema]]
+    field = "notification_sound"
+    value = ""
 ```
```

Comment:
Just an alternative, see which looks better:

```
  config_schema = [
    {field = "roomname", value = "The room"},
    {field = "subject", value = "Chit-chat"},
    {field = "background", value = ""},
    {field = "notification_sound", value = ""}
  ]
```

Btw, how do we eliminate `default_config`? it seems to be a config option created internally by muc_light, I wonder why it was in the example then.

Reply:
Thanks for the suggestion, it looks better I think, but I've made a big mistake. From what I'm reading now, the whole section doesn't make any sense as described now - without the `default_config` option (as is in MIM from 3.6) the problem described in this paragraph will not appear. So it's completely senseless to describe it with TOML config as this problem only appears in older MIMs. I will revert changes to this file because of that.

Reply:
I think it's correct now that it's reverted.